components/EntityAttributeValue.js

```diff
@@ -32,5 +34,10 @@ const formatImage = (key) => {
       case 'float':
         //TODO: Find a more accurate representation of float
         return Number(item.value_double);
+      case 'custom':
+        const Data = plugin(item.attributes_custom_name);
+        const data = new Data();
+
+        return data.toApi(item.attributes_custom_name, item.value_id, item.value_long_string);
```

Comment:
Eto pala tinatanong mo kanina
```suggestion
        const CustomAttributeType = plugin(item.attributes_custom_name);
        const customAttributeType = new CustomAttributeType();

        return customAttributeType.toApi(item.attributes_custom_name, item.value_id, item.value_long_string);
```